test/e2e/helpers.js

```diff
@@ -898,7 +866,6 @@ module.exports = {
   roundToXDecimalPlaces,
   generateRandNumBetween,
   sleepSeconds,
```

Comment:
Can we also remove `sleepSeconds`, it's only used in `test/e2e/mv3/multiple-restarts.spec.js` which is being deleted here. It's also a duplicate of `await driver.delay()`

Reply:
Ops, will catch in another MR !